src/style.css

```
@@ -1,3 +1,69 @@
+
 body {
   margin: 0;
+background-color: #fff;
+font-family: Roboto, sans-serif, Arial;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+display: block;
+line-height: 0;
+}
+
+.header {
+box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+background: #FFF;
+}
```

Need to fix code style everywhere
```suggestion
.header {
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
  background: #FFF;
}
```